src/components/DataDecorator/Decorator/Decorator.js

```diff
@@ -71,29 +74,28 @@ class Decorator extends Component {
       scoreThresholds
     );
     const decoratorClasses = classnames(namespace, classes, className, {
-      [`${namespace}--link`]: this.props.href,
-      [`${namespace}--active`]: this.props.active,
-      [`${namespace}--inert`]: this.props.inert,
-      [`${namespace}--inline`]: this.props.inline,
+      [`${namespace}--link`]: href,
+      [`${namespace}--button`]: onClick,
+      [`${namespace}--active`]: active,
+      [`${namespace}--inert`]: !href && !onClick,
+      [`${namespace}--inline`]: inline,
     });
 
-    if (this.props.href) {
+    if (href) {
       return (
-        <Link href={this.props.href} className={decoratorClasses} tabIndex={0}>
+        <Link href={href} className={decoratorClasses} tabIndex={0}>
           {decorator}
         </Link>
       );
+    } else if (onClick) {
+      return (
+        <button className={decoratorClasses} onClick={this.handleClick}>
+          {decorator}
+        </button>
+      );
     }
 
-    if (inert) {
-      return <span className={decoratorClasses}>{decorator}</span>;
-    }
-
-    return (
-      <button className={decoratorClasses} onClick={this.handleClick}>
-        {decorator}
-      </button>
-    );
+    return <span className={decoratorClasses}>{decorator}</span>;
```

Comment:
"inert" span is now the default. If `href` is provided, we render a link. If `onClick` is provided, we render a button.